assets/javascripts/discourse/templates/list/topic-list-item.raw.hbs

```diff
@@ -46,13 +55,20 @@
   </div>
 
   {{#if tilesOrSocial}}
+    {{#unless tilesStyle}}
     <div class="main-link-footer">
+      {{raw-plugin-outlet name="topic-list-main-link-footer"}}
+    {{/unless}}
+      {{raw 'list/topic-meta' tilesStyle=tilesStyle likesHeat=likesHeat title=view.title topic=topic}}
       {{#if showActions}}
+        {{#if tilesStyle}}
+          </div>
```

Comment:
This nested block of conditionals (57 to 72) makes me nervous. Particularly because we have a single closing div wrapped in a conditional. This is a red flag. Try to move some of this conditional logic to the component. As far as possible, try to keep logic in the component. I don't always do that myself, but it is better practice and needed in this case.

Reply:
I've reverted this file to the current HEAD. All layout changes are now done with jQuery. A little worried about performance now though.